add example @now/node example using express (#651)
zeit/docs: pages/docs/v2/deployments/official-builders/node-js-now-node.mdx

For each invocation of a Node.js lambda, two objects, request and response, are passed to the function. These objects are the standard HTTP [request](https://nodejs.org/api/http.html#http_event_request) and [response](https://nodejs.org/api/http.html#http_class_http_serverresponse) objects given and used by Node.js.

> Because many web serving frameworks like [express](http://expressjs.com/) also frequently deal with standard HTTP request and response objects, you may be able to use them in your Node.js lambda. The following lambda accepts a JSON body in the POST request, and returns a stringified version of it.

Reviewer: ```suggestion
With many Node.js-based frameworks like [Express](http://expressjs.com/) also frequently dealing with these HTTP request and response objects, you may be able to use them in your Node.js lambda. The following lambda example, using Express, accepts a JSON body in the POST request, and returns a stringified version of it.
```

<Example>
  <span>Example Node.js lambda using express and body-parser</span>
  <Code lang="javascript">{`'use strict'\n
const express = require('express')
const bodyParser = require('body-parser')\n
const app = express()
module.exports = app\n
app.use(bodyParser.json())\n
app.post('*', (req, res) => {
  if (req.body == null) {
    return res.send(400, { error: 'no JSON object in the request' })
  }\n
  res.set('Content-Type', 'application/json')
  res.send(200, JSON.stringify(req.body, null, 4))
})\n
app.all('*', (req, res) => {
  res.send(405, { error: 'only POST requests are accepted' })
})`}</Code>
</Example>

> For a further example, see the blog post [Serverless Express.js with Now 2](https://zeit.co/blog/serverless-express-js-lambdas-with-now-2).

Reviewer: I'd prefer if we linked to an example that can be updated:
```suggestion
For more information, see the following example: [Serverless Express with Now 2](/examples/express/).
```

Author: yeah, that will be better; in that case, the example should be a little more interesting, like using body-parser.  Also, the [`now.json` content in the README](https://github.com/zeit/now-examples/tree/master/express) doesn't match the [`now.json` file at master](https://github.com/zeit/now-examples/blob/master/express/now.json).

I'll file a separate issue on that.

Author: And actually the https://github.com/zeit/now-examples/tree/master/express-twitter-dreamify example of non-trivial express usage, eg, using middleware.
